tests/makecolumns.sil

```diff
@@ -0,0 +1,5 @@
+\begin[papersize=a5]{document}
+\script[src=packages/frametricks]
```

Comment:
@jodros I'll fix this test, but just for your own reference in the future, the non-deprecated syntax for this is:

```suggestion
\script[src=packages/frametricks]
\use[module=packages.frametricks]
```